packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -108,57 +109,58 @@ export class BlobManager {
         // When this blob is retrieved, let the container runtime know that the corresponding GC node got updated.
         this.gcNodeUpdated(this.getBlobGCNodePath(blobId));
 
-        return new BlobHandle(
-            `${BlobManager.basePath}/${storageId}`,
-            this.routeContext,
+        return PerformanceEvent.timedExecAsync(
```

Comment:
This will not measure readBlob() call, it will measure how long it takes to run BlobHandle() constructor, which is not what we want.